utils/lv2-ttl-generator/lv2_ttl_generator.c

```diff
@@ -62,31 +68,27 @@ int main(int argc, char* argv[])
 
     if (ttlFn != NULL)
     {
-        char basename[strlen(argv[1])+1];
+        char* normalPath = makeNormalPath(path);
```

Comment:
It is not easy to follow what all this is doing, some code comments would be welcome